src/index.ts

```diff
@@ -49,14 +48,14 @@ const {grpc} = new gax.GrpcClient();
 export interface GetInstancesCallback {
   (
     err: ServiceError | null,
-    result?: Instance[] | null,
-    nextQuery?: {} | null,
-    response?: google.bigtable.admin.v2.IListInstancesResponse | null
+    result?: Instance[],
+    failedLocations?: string[],
```

Comment:
BREAKING CHANGE
exposing [`failedLocations`](https://github.com/googleapis/googleapis/blob/0d7c3565dd942d0cc41b16e895046486ae29e44e/google/bigtable/admin/v2/bigtable_instance_admin.proto#L305) in the callback/response

Reply:
Let's make sure we add docs for this new response argument.

Reply:
Also, an example in the code block showing "if (failedLocations.length > 0) { // These locations contain instances which could not be retrieved. }"